src/util.star

```diff
@@ -37,3 +37,7 @@ def to_hex_chain_id(chain_id):
     out = "%x" % int(chain_id)
     pad = 64 - len(out)
     return "0x" + "0" * pad + out
+
+
+def multiline_cmd(commands):
+    return " && ".join(commands)
```

Comment:
Technically this is multiple complete commands which have been joined using `&&`, right? I can't take
```
ls
-al
/
```
And process that via `multiline_cmd`, because it would insert `&&`.

Suggest to rename this function `join_cmds` to make it clear that each line must still be a complete command of its own.

Reply:
that's a fair point, i'll make that change